server/services/group.js

```diff
@@ -31,4 +31,33 @@ function updateGroup(groupName, groupObject) {
       Group.findOneAndUpdate({ name: groupName }, validatedGroupObject, { new: true }));
 }
 
-export default { getGroup, deleteGroup, createGroup, updateGroup };
+function addUser(group, userId) {
+  const userIdSchema = Joi.string().hex().length(24);
+
+  return JoiHelper.validate(userId, userIdSchema)
+    .then((validatedUserObject) => {
+      group.addUser(validatedUserObject);
+      return group;
```

Comment:
This is in multiple places around the codebase, we should do the others not touched by this PR in a different PR